### DISCUSSION TITLE
Regarding file transfer from phone to windows 10 issue

### DISCUSSION THREAD

**Discussion by Rishabh224** [localsend/localsend#1223] [Discussion (English, International)]
I have tried multiple times but whenever i want to transfer any data from my phone to laptop windows 10 and i click the button on send from phone i don't get any message prompt of accept or receive on my laptop. But vice versa is happening when i transfer anything from my laptop to phone the prompt shows up in my phone to accept or reject the transfer. Please help how to resolve this issue?????

**Comment by CarpeSergio** [comment]
Yes! It happened the same for me and the solution is very very simple. A lot of time to discover this haha. You must go to the Internet Options and enable the Private Network instead of the Public Network settings to be able to share files within your local devices. If you have enabled the Private one, you need to do the same in the Firewall Windows Defender 
![ssssssssssssss](https://github.com/localsend/localsend/assets/169681739/61b301ee-27bf-445f-87a1-5882ab909a8f)

**Reply by lmcorr** [reply]
This fixed the issue for me thank you 👍
